README.md

```diff
@@ -1,37 +1,38 @@
 # FinOps Foundation SIG Containers
 
-Primary Authors: tbc
+Primary Authors: Mike Fuller
 
 Reviewed and Contributed to by: tbc
 
 # Introduction
 
-tbc
+SIG-Containers exists to explore how FinOps applies to the container world within cloud. To help identify/define the added cloud financial management challenges that containers introduce and solutions/best practices around how to apply FinOps to solve them. 
```

Comment:
I like this, I would just break apart and clarify the second sentence.
" Containers are a fast-growing space that changes rapidly but has tremendous potential benefits. The goal is to define the additional cloud financial management challenges that come with containers while providing solutions and best practices for how to proactively address those challenges."